src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+//import edu.wpi.first.wpilibj2.*;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.MathUtil;
+
+
+
+/** Add your docs here. */
+public class LimelightSubsystem extends SubsystemBase {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+  private PIDController m_PidController;
+  private DriveBaseSubsystem m_DriveBaseSubsystem;
+  private CANSparkMax m_leftMotor;
+  private CANSparkMax m_rightMotor;
+  private boolean isFinished;
+
+  public LimelightSubsystem() {
+    m_PidController = new PIDController(6e-3, 0, 0);
+    m_PidController.setTolerance(2.0);
+    m_DriveBaseSubsystem = RobotContainer.getDriveBase();
+    m_leftMotor = m_DriveBaseSubsystem.getLeftMotor();
+    m_rightMotor = m_DriveBaseSubsystem.getRightMotor();
```

Comment:
Instead of having to deal with the motors directly here, it would be better to add a setSpeed method to Drivebasesubsystem and call that. 

Then if there's changes to the motor configuration it doesn't mess up your code